src/js/models/layer.js

```diff
@@ -288,7 +288,8 @@ define(function (require, exports, module) {
                 isLinked: this.isLinked,
                 vectorMaskEnabled: this.vectorMaskEnabled,
                 vectorMaskEmpty: this.vectorMaskEmpty,
-                textWarningLevel: this.textWarningLevel
+                textWarningLevel: this.textWarningLevel,
+                smartObject: this.smartObject
```

Comment:
having a hard time figuring out how this change is required for this PR.  I assume there's a reason.